src/components/ordered-list-item/OrderedListItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode  } from 'react'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import { SxProps } from '@mui/material'
+
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+
+import { styles } from '~/components/ordered-list-item/OrderedListItem.styles'
+
+type OrderedListItemProps = {
+  title: ReactNode
```

Comment:
The title must be a string, a description I think the same

Reply:
yep, u r right, fixed